src/csharp/Microsoft.Spark/Sql/Types/DataType.cs

```diff
@@ -67,7 +67,21 @@ public string Json
         /// JSON value of this data type.
         /// </summary>
         internal virtual object JsonValue => TypeName;
-
+        
+        /// <summary>
+        /// Parses a JSON string to create a JvmObjectReference object.
```

Comment:
, which references a StructType object on the JVM side.

Reply:
Sure, added.